src/components/hooks/use-academy-data.tsx

```diff
@@ -69,7 +69,7 @@ const query = graphql`
                     description
                     imageFile {
                         childImageSharp {
-                            gatsbyImageData(width: 600, aspectRatio: 1.725)
+                            gatsbyImageData(width: 600, aspectRatio: 1.82)
```

Comment:
Why do we have this change in this PR?

Reply:
To fix vercel issue and check the task is complete. Without this we can't get test link due to error in vercel